a3p-integration/proposals/a:vaults-auctions/upgradeVaults.test.js

```diff
@@ -103,6 +107,22 @@ const verifyVaultPriceUpdate = async t => {
   t.is(quote.value[0].amountOut.value, '+5200000');
 };
 
+const verifyAuctionInstance = async t => {
+  const newAuctionInstance = await getAuctionInstance();
+  const oldInstance = await readFile(
+    `${env.HOME}/.agoric/previousInstance.json`,
+    'utf-8',
+  );
+
+  console.log(
+    `new: ${newAuctionInstance} should be different from ${oldInstance}`,
+  );
+  t.true(
+    newAuctionInstance !== oldInstance,
```

Comment:
Probably not worth a re-spin for this alone, but for future ref:

If you use `t.not`, ava will show the 2 values when the assertion fails.

```suggestion
  t.not(newAuctionInstance,
          oldInstance,
```